server/tests-py/test_graphql_queries.py

```diff
@@ -263,6 +263,7 @@ def test_staff_passed_students(self, hge_ctx, transport):
     def test_user_query_auction(self, hge_ctx, transport):
         check_query_f(hge_ctx, self.dir() + '/user_query_auction.yaml', transport)
 
+    @pytest.mark.xfail(reason="Query caching")
```

Comment:
Could you comment with a link back to the open issue: https://github.com/hasura/graphql-engine-internal/issues/252

Reply:
Reference to the above mentioned issue has been provided.